src/cmd/services/m3query/config/config.go

```diff
@@ -146,6 +151,22 @@ type FilterConfiguration struct {
 	CompleteTags Filter `yaml:"completeTags"`
 }
 
+// ResultOptions are the result options for query.
+type ResultOptions struct {
+	// DropNaNsInResults drops NaNs before returning query results.
+	// The default is true, which matches Prometheus
+	DropNaNsInResults *bool `yaml:"dropNaNs"`
```

Comment:
Also can you call it `dropNans` instead, the NaN is annoying and out of place in a config